File: Java_Programming/Access.cpp
#include <iostream>
using namespace std;

class Demo
{
    public :
        int i;
    private:
        int j;
    protected:
        int k;


    public:
        Demo()
        {
            i = 0;
            j = 0;
            k = 0;
        }

        void Display()
        {
            cout<<"Valuse of i = "<<i<<"\n";   // allowed
            cout<<"Valuse of j = "<<j<<"\n";  // allowed
            cout<<"Valuse of k = "<<k<<"\n";   // allowed
        }

};


int main()
{
    Demo dobj;
    dobj.Display();

    cout<<"Valuse of i = "<<dobj.i<<"\n";   //allowed
    cout<<"Valuse of j = "<<dobj.j<<"\n";   //not allowed
    cout<<"Valuse of k = "<<dobj.k<<"\n";    // not allowed

    return 0;
}

File: Java_Programming/Access.java
class Demo
{
    public int i;
    private int j;
    protected int k;

    public Demo()
    {
        i =0;
        j=0;
        k=0;
    }

    public void Display()
    {
        System.out.println("Value of i="+i);   //allowed
        System.out.println("Value of i="+j);  //allowed
        System.out.println("Value of i="+k);  //allowed

    }
}

class Access
{
    public static void main(String A[])
    {
        Demo dobj = new Demo();

        dobj.Display();

        System.out.println("Value of i="+dobj.i);  // allowed
        System.out.println("Value of i="+dobj.j);   //not allowed
        System.out.println("Value of i="+dobj.k);   // ???????

    }
}